unittest/resources/test_user.py

```diff
@@ -26,14 +27,196 @@ def test_something(self):
 
         assert resp.status_code == 200
         assert json.loads(resp.data) == {
-            'info': None,
+            'info': {},
+            'status': {
+                'statusMsg': 'Ok',
+                'statusDetails': {},
+                'statusCode': 'HTTPOK'
+            },
+            'result': {
+                'id': 1,
+                'name': 'Test User',
+                'email': 'test@test.com'
+            }
+        }
+
+    def test_get_notFound(self):
+        """Verify NotFound is raised when userId does not exist
+        """
+        resp = self.client.get('/users/1000')
+
+        assert resp.status_code == 404
+        assert json.loads(resp.data) == {
+            'info': {},
+            'status': {
+                'statusCode': 404,
+                'statusMsg': 'Not found',
+                'statusDetails': {}
+            },
+            'result': {}
+        }
+
+    def test_get_list_success(self):
+        """Test list of users is successfully returned.
+        """
+        user1 = User('Test1 User', 'test1@test.com', 'password1')
+        user2 = User('Test2 User', 'test2@test.com', 'password2')
+        db.session.add(user1)
+        db.session.add(user2)
+        db.session.commit()
+        resp = self.client.get('/users')
+
+        assert resp.status_code == 200
+        assert json.loads(resp.data) == {
+            'info': {},
             'status': {
                 'statusMsg': 'Ok',
                 'statusDetails': {},
                 'statusCode': 'HTTPOK'
             },
+            'result': [{
+                'id': 1,
+                'name': 'Test1 User',
+                'email': 'test1@test.com'
+                },
+                {
+                'id': 2,
+                'name': 'Test2 User',
+                'email': 'test2@test.com'
+                }]
+        }
+
+
+class TestUser_post(TestUser):
+    """Test resources for the API post endpoint.
+    """
+    def test_post_success(self):
+        user = {
+            'name': 'Test User',
+            'email': 'test@test.com',
+            'password': 'test'
+            }
+
+        resp = self.client.post('/users/', data=user)
+
+        assert resp.status_code == 200
+        assert json.loads(resp.data) == {
+            'info': {},
+            'status': {
+                'statusCode': 'HTTPOK',
+                'statusMsg': 'Ok',
+                'statusDetails': {}
+            },
+            'result': {
+                'id': 1,
+                'name': 'Test User',
+                'email': 'test@test.com'
+            }
+        }
+
+    def test_post_badRequest(self):
+        user = {
+            'name': None,
+            'email': None,
+            'password': None
+        }
```

Comment:
Since keys with None are dropped during the request, you can just make this an empty dict.